pkg/virt-operator/install-strategy/create.go

```diff
@@ -1425,6 +1426,57 @@ func addOrRemoveSSC(targetStrategy *InstallStrategy,
 	return nil
 }
 
+func createOrUpdateSCC(kv *v1.KubeVirt,
+	targetStrategy *InstallStrategy,
+	stores util.Stores,
+	clientset kubecli.KubevirtClient,
+	expectations *util.Expectations) error {
+	sec := clientset.SecClient()
+
+	if !stores.IsOnOpenshift {
+		return nil
+	}
+
+	version := kv.Status.TargetKubeVirtVersion
+	imageRegistry := kv.Status.TargetKubeVirtRegistry
+	id := kv.Status.TargetDeploymentID
+	kvkey, err := controller.KeyFunc(kv)
+	if err != nil {
+		return err
+	}
+
+	for _, scc := range targetStrategy.sccs {
+		var cachedSCC *secv1.SecurityContextConstraints
+		scc := scc.DeepCopy()
+		obj, exists, _ := stores.SCCCache.GetByKey(scc.Name)
+		if exists {
+			cachedSCC = obj.(*secv1.SecurityContextConstraints)
+		}
+
+		injectOperatorMetadata(kv, &scc.ObjectMeta, version, imageRegistry, id)
+		if !exists {
+			expectations.SCC.RaiseExpectations(kvkey, 1, 0)
+			_, err := sec.SecurityContextConstraints().Create(scc)
+			if err != nil {
+				expectations.SCC.LowerExpectations(kvkey, 1, 0)
+				return fmt.Errorf("unable to create SCC %+v: %v", scc, err)
+			}
+			log.Log.V(2).Infof("SCC %v created", scc.Name)
+		} else if !objectMatchesVersion(&cachedSCC.ObjectMeta, version, imageRegistry, id) {
+			_, err = sec.SecurityContextConstraints().Update(scc)
```

Comment:
I'm not 100% sure if this Update will always work unless we copy the metadata from the old cachedSCC and then override only the version/registry annotations.

So, something like this. 

```
scc.ObjectMeta = cachedSCC.ObjectMeta.DeepCopy()
injectOperatorMetadata(kv, &scc.ObjectMeta, version, imageRegistry, id)
_, err = sec.SecurityContextConstraints().Update(scc)
if err != nil {
	return fmt.Errorf("Unable to update %s SecurityContextConstraints", scc.Name)
}
log.Log.V(2).Infof("SecurityContextConstraints %s updated", scc.Name)
```

Reply:
ah yes, you pointed out that already in previous comments, sorry, I will fix it.

Reply:
done